pom.xml

```diff
@@ -67,6 +67,21 @@
     -->
   </dependencies>
   
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <version>3.0.1</version>
+        <configuration>
+          <xmlOutput>true</xmlOutput>
+          <findbugsXmlWithMessages>true</findbugsXmlWithMessages>

```

Comment:
Depends on the Maven profile. If we ant to run it in all cases, we need to explicitly declare it